schemas/WebpackOptions.json

```
@@ -133,6 +136,19 @@
         }
       ]
     },
+    "AssetModulePublicPath": {
+      "description": "The publicPath of asset modules.",
+      "anyOf": [
+        {
+          "type": "string",
+          "absolutePath": false
```

```suggestion
          "type": "string"
```

paths like `/assets/` will look like absolute paths and would fail the validation